components/carte-etat/carte-etat-component.jsx

```diff
@@ -140,62 +138,58 @@ class CarteEtat extends PureComponent {
+              <CircularProgress color="secondary" />
+            </center>
+          )}
+          {!isDisabledEtat && !isLoadingEtat && (
+            <div>
+              <div className="chart-wrapper">
+                <div className="main-chart">
+                  <BarChart deciles={deciles} />
+                </div>
+                <div className={classes.simpop}>
+                  <div className="montant-impots">
+                    <Typography inline className="impotEtat avant">
+                      {totalAvant}
+                    </Typography>
+                    <Typography inline className="uniteImpotEtat avant">
                           Md€*
-                        </Typography>
-                      </div>
-                      <div className="montant-impots">
-                        <Typography inline className="impotEtat plf">
-                          {totalPLF}
-                        </Typography>
-                        <Typography inline className="uniteImpotEtat plf">
+                    </Typography>
+                  </div>
+                  <div className="montant-impots">
+                    <Typography inline className="impotEtat plf">
+                      {totalPLF}
+                    </Typography>
+                    <Typography inline className="uniteImpotEtat plf">
                           Md€*
-                        </Typography>
-                      </div>
-                      <div className="montant-impots">
-                        <Typography inline className="impotEtat apres">
-                          {totalApres}
-                        </Typography>
-                        <Typography inline className="uniteImpotEtat apres">
+                    </Typography>
+                  </div>
+                  <div className="montant-impots">
+                    <Typography inline className="impotEtat apres">
+                      {totalApres}
+                    </Typography>
+                    <Typography inline className="uniteImpotEtat apres">
                           Md€*
-                        </Typography>
-                      </div>
-                    </div>
+                    </Typography>
                   </div>
-                  <Typography className={classes.sourceInsee}>
+                </div>
+              </div>
+              <Typography className={classes.sourceInsee}>
                     * Chiffrages indicatifs.
-                    <br />
-                    {" "}
+                <br />
+                {" "}
 Estimation à partir des données de l&apos;Enquête
                     Revenus Fiscaux et Sociaux (ERFS-FPR) de l&apos;Insee.
-                  </Typography>
-                </div>
-              ),
-            ]
+              </Typography>
+            </div>
           )}
         </CardContent>
         { (isLoadingEtat || isDisabledEtat) ? ("")
           : (
-            <ExpansionPanel
```

Comment:
`isPanelExpanded` and `expandArticlePanelHandler` were undefined so I removed them. In other components of the application, these properties are used to allow only one expanded panel at a time.